schema/patches/3.sql

```diff
@@ -42,7 +47,9 @@ AS $$
                       (effective_to >= NOW() OR effective_to IS NULL)
         );
 
-        -- Right, we've got what we need
+        -- Right, we've got what we need so dump output into analytics table
+	--  Preserve reference data (like session_id, current datetime) for comparison
+	--  TODO Explain why num_shows is rounded in that way
```

Comment:
Anonymity

Reply:
Does nothing else in the analytics show which member it's referencing? Also what is this actually used for?

Reply:
The idea of the design is that an individual analytics record wouldn’t identify any one member (you can guess based on officerships, but for people without officerships all you get is the number of shows to the nearest 5). I put it in when I was planning myradio2 (lol) to see which pages what “kinds of people” used more. I never removed it - though it might be disabled on prod now.

Reply:
Right that explains why it's not related to the stats pages then. I've added a brief bit to take note of that, even if it's not strictly being used. Is the rest of the PR alright?